.travis.yml

```diff
@@ -1,5 +1,13 @@
 ---
 language: ruby
-rvm: 2.1.2
-before_script: bundle exec appraisal install
-script: bundle exec appraisal rspec
+rvm:
+  - 2.0.0
```

Comment:
My original goal here was to run tests against the versions of Ruby that Rails 4.1 and 4.2 support, but I think it's reasonable to drop anything < 2.1.